wordpress-mobile/WordPress-iOS
# Fix layout of the cell and use standard cell.

Fixes #8173 

To test:
 - Create a new post
 - Go to Post Options
 - Tap on Category
 - Tap on +
 - Check that the label for the category title shows correctly.
